### ISSUE TITLE
NFL Game Info and player salary?

### ISSUE THREAD

**Issue by bpisaacoff** [roclark/sportsreference#227]
Is there a way to access the Game Info for NFL games (see for example https://www.pro-football-reference.com/boxscores/201810070cin.htm) which has info like roof and weather and vegas line. 

Additionally is there a way to access player (and possibly coach) salary?

Thanks,

**Comment by roclark** [comment]
Hey @bpisaacoff, thanks for the suggestions! As you already gathered, those features are not presently included with the package, but I would be very happy to include them with a future release! My next non-patch release (0.5.0) is currently shaping up to add a bunch of features like these, so I think it makes sense to include them there. For easier tracking, I might split these core features into separate issues as it makes it easier on my side.

Thanks again! These are great ideas and will be quite useful!

**Comment by roclark** [comment]
Hey @bpisaacoff, I realize my response here is nearly a year old at this point, but just wanted to follow-up that I added the NFL game info yesterday and will include it with the next release of `sportsreference`. Additionally, I've been tracking the salary info for NFL, and I found that the sports-reference.com team doesn't really maintain player salaries anymore, and all links to the player salaries are broken. At this point, since the other main part of this issue is merged, I think I will go ahead and close this issue and mark it as complete. If you find that there is a good, reliable location on sports-reference.com which has player salary, I would be very happy to include that with the package! Just fire up another issue with a link to a page I can scrape, and I can take care of that.

Thanks again for the feedback, and I apologize for the very overdue response!
